litmus-portal/frontend/src/pages/WorkflowInfoStats/StackedBar/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { useQuery } from '@apollo/client';
+import { Typography, useTheme } from '@material-ui/core';
+import { LineMetricSeries, StackBar, StackBarMetric } from 'litmus-ui';
+import moment from 'moment';
+import React, { useState } from 'react';
+import Loader from '../../../components/Loader';
+import Center from '../../../containers/layouts/Center';
+import { WORKFLOW_DETAILS } from '../../../graphql';
+import {
+  Workflow,
+  WorkflowDataVars,
+} from '../../../models/graphql/workflowData';
+import { getProjectID } from '../../../utils/getSearchParams';
+import { STATUS_RUNNING } from '../../ApplicationDashboard/constants';
+import WorkflowRunTable from '../WorkflowRunTable';
+import useStyles from './styles';
+
+interface StackedBarGraphProps {
+  date: number;
+  workflowID: string;
+  handleTableOpen: () => void;
+  handleTableClose: () => void;
+  showTable: boolean;
+}
+const StackedBarGraph: React.FC<StackedBarGraphProps> = ({
+  date,
+  workflowID,
+  handleTableOpen,
+  handleTableClose,
+  showTable,
+}) => {
+  const projectID = getProjectID();
+  const classes = useStyles();
+  const theme = useTheme();
+  const [workflowRunID, setWorkflowRunID] = useState<string>('');
+
+  const stackBarData: Array<StackBarMetric> = [];
+
+  const openSeries: LineMetricSeries = {
+    metricName: 'Resiliency Score',
+    data: [],
+    baseColor: '#5469D4',
+  };
+
+  const { data, loading } = useQuery<Workflow, WorkflowDataVars>(
+    WORKFLOW_DETAILS,
+    {
+      variables: {
+        workflowRunsInput: {
+          project_id: projectID,
+          workflow_ids: [workflowID],
+          sort: {
+            field: 'Time',
+          },
+          filter: {
+            date_range: {
+              start_date: moment.unix(date).startOf('day').unix().toString(),
+              end_date: moment.unix(date).endOf('day').unix().toString(),
+            },
+          },
+        },
+      },
+      fetchPolicy: 'cache-and-network',
+    }
+  );
+
+  if (data?.getWorkflowRuns.workflow_runs) {
+    data.getWorkflowRuns.workflow_runs.forEach((wfrun) => {
```

Comment:
```suggestion
    data?.getWorkflowRuns?.workflow_runs?.forEach((wfrun) => {
```
and the if can be removed